src/upcoming-components/Textarea/Textarea.tsx

```diff
@@ -96,12 +96,7 @@ export const Textarea = ({
   defaultValue,
   ...other
 }: Props) => {
-  const componentClassName = clsx(
-    styles['textarea'],
-    className,
-    disabled && styles['eds-is-disabled'],
-    isError && styles['eds-is-error'],
```

Comment:
Why don't we need error and disabled styling on this anymore?

Reply:
Oh nvm you're handling that in the mixin now. Then my question is whether we need to do something with the `isError` and `disabled` props being deconstructed just above here.

Reply:
We probably do need error and disabled styling on this, will re-append the `error` style but with the syntax here (i.e. `isError && styles['error']`. Disabled will be handled via the pseudoclass `:disabled` so doesn't need a class attached. 

The error/disabled styling in the mixin is not part of this pr, was happening already, just renaming it so it's not `eds-is-error` and `eds-is-disabled`

Reply:
Yup that worked:

![Screen Shot 2022-08-26 at 9 53 22 AM](https://user-images.githubusercontent.com/86632227/186954590-07b1d1fd-ba53-4c04-81e3-c72e9ce5b572.png)
![Screen Shot 2022-08-26 at 9 53 26 AM](https://user-images.githubusercontent.com/86632227/186954592-f5d75864-aa0c-4c59-832e-0ec76e2e3793.png)

Reply:
Nice!